_maps/skyrat/automapper/templates/birdshot/birdshot_blueshield.dmm

```diff
@@ -0,0 +1,23 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/obj/structure/cable,
+/obj/structure/disposalpipe/segment{
+	dir = 4
+	},
+/obj/effect/turf_decal/siding/thinplating_new/terracotta{
+	dir = 4
+	},
+/obj/structure/closet/secure_closet/blueshield,
```

Comment:
This is already spawned in by the AM SS, or should be at least.

[Code](https://github.com/Skyrat-SS13/Skyrat-tg/blob/5dcdaaa97393e5baa603cd44ecbcb711351f1bc5/modular_skyrat/modules/automapper/code/area_spawn_entries.dm#L23)

```dm
/datum/area_spawn/blueshield_locker
	target_areas = list(/area/station/command/heads_quarters/captain, /area/station/command/bridge)
	desired_atom = /obj/structure/closet/secure_closet/blueshield
	mode = AREA_SPAWN_MODE_HUG_WALL
````

Reply:
I'm aware, there is no valid location for the AM spawner, all the walls are taken.